ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml

```
@@ -0,0 +1,58 @@
+base_images:
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: jaeger-product-rosa-hypershift
+  cron: 0 10 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.13"
+      OPERATORS: |
+        [
+            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"}
```

I can't confirm or deny at the moment that what you have here will work we'll see what happens in the rehearsal job but I have tested this config for the operator install and it did work

```{"name": "jaeger-product", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-distributed-tracing", "operator_group": "jaeger-operator-group"}```

---

@ascerra I think this should work. But I was bit confused about why PQE is installing the operator this way
```suggestion
            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"}https://github.com/openshift/release/pull/40442/commits/2c27c5d036950e2611c29be9d909914531b07de0#diff-112ea49c1cd2a5af172ba25b6de85ab5679955bc55814e9cd3f3113428cb47a6
```
for onboarding